rfcs/text/0000-authentication-fields.md

```diff
@@ -0,0 +1,155 @@
+
+## Fields
+
+#### Activity Log
+|Proposed Field Name|Type|Value|Azure Field|
+| --- | --- | --- | --- |
+|authentication.scope|keyword|user_impersonation|azure.activitylogs.identity.claims.http://schemas_microsoft_com/identity/claims/scope|
+|authentication.authentication.requirement|keyword|multiFactorAuthentication|azure.activitylogs.properties.authenticationRequirement|
+|authentication.additional.details|keyword|MFA requirement satisfied by claim in the token|azure.activitylogs.properties.status.additionalDetails|
+|authentication.token.type|keyword|AzureAD|azure.activitylogs.properties.tokenIssuerType|
+|authentication.token.identifier|keyword|MjlkY2M0MzItNWU4YS00NjU5LTlmMDMtNmVkZTE4NDAwMzAw|azure.activitylogs.properties.uniqueTokenIdentifier|
+|authentication.authorization|keyword|ROOTMANAGESHAREDACCESSKEY|azure.resource.authorization_rule|
+|authentication.isInteractive|keyword|FALSE|azure.signinlogs.properties.is_interactive|
+|authentication.token|keyword|AzureAD|azure.signinlogs.properties.token_issuer_type|
+
+
+#### SignIn Log
+|Proposed Field Name|Type|Value|Azure Field|
+| --- | --- | --- | --- |
+|authentication.authentication.requirement|keyword|singleFactorAuthentication|azure.signinlogs.properties.authentication_requirement|
+|authentication.isInteractive|boolean|FALSE|azure.signinlogs.properties.is_interactive|
+|authentication.risk|keyword|low|azure.signinlogs.properties.risk_level_aggregated|
+|authentication.token.type|keyword|AzureAD|azure.signinlogs.properties.token_issuer_type|
+|authentication.token.identifier|keyword|MDg4YjQ0MDktOWU2My00MjVkLWI3NzctMmM4YzZjMzgwYjAw|azure.signinlogs.properties.unique_token_identifier|
+
+## Usage
+| Field | Description | Example Values(s) |
+| --- | --- | --- |
+|authentication.additional.details|Login Additional Details|MFA requirement satisfied by claim in the token|
+|authentication.authentication.requirement|Login Authentication Requirements|singleFactorAuthentication/multiFactorAuthentication|
+|authentication.authorization|Login Authorization Information|ROOTMANAGESHAREDACCESSKEY|
+|authentication.isInteractive|Is the login an interactive login?|TRUE/FALSE|
+|authentication.risk|Risk level for the login event.|low|
+|authentication.scope|Requested scope of user|user_impersonation|
+|authentication.token.identifier|Login token identifier/value|MjlkY2M0MzItNWU4YS00NjU5LTlmMDMtNmVkZTE4NDAwMzAw|
+|authentication.token.type|Type of token for the login event|AzureAD|
+|authentication.id|Unique ID given across primary authentication and additional (multi factor) authentication.||
+|authentication.protocol|Authentication Protocol (authproto)||
+
+
+## Source data
+
+The source data for the logn fields came from Azure data, which came from Azure Eventhubs, or Blob Storage. The specific types of data being pulled from these sources are Signin, Platform, Activity, and Audit logs.
```

Comment:
I think it's great that you have a example dataset for these fields, but have you taken a look at any other vendors/products login / auth logs?

I believe we discussed this on the former PR, but it's definitely important to make sure that we have broad coverage instead of being too specific to one thing.

Reply:
@kgeller for sure. The Palo logs do fit in with what we have, but will take another look.

Reply:
Added in the Palo Alto logs and made it more vendor neutral